modules/microshift-building-apps-rpms.adoc

```diff
@@ -0,0 +1,63 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-building-apps-rpms_{context}"]
+= Building the RPM package for the application manifests
+
+To building your own RPMs, you must create a `.spec` file that adds the application manifests to the RPM package. The following is an example procedure. As long as the application RPMs and other elements needed for image building are accessible to Image Builder, you can use the method you prefer.
+
+.Prerequisites
+* You have set up a Red Hat Enterprise Linux (RHEL) for Edge 9.2 build host that meets the Image Builder system requirements.
+* You have root access to the host.
+* The file tree required to build RPM packages was created.
+
+.Procedure
+
+. In the `~/rpmbuild/SPECS` directory, create a file named `<application-workload-manifests.spec>` using the following example content:
++
+.Example `.spec` file
++
+[source,terminal]
+----
+Name: application-workload-manifests
+Version: 0.0.1
+Release: 1%{?dist}
+Summary: Adds workload manifests to microshift
+BuildArch: noarch
+License: GPL
+Source0: %{name}-%{version}.tar.gz
+#Requires: microshift
+%description
+Adds workload manifests to microshift
+%prep
+%autosetup
+%install <1>
+rm -rf $RPM_BUILD_ROOT
+mkdir -p $RPM_BUILD_ROOT/%{_prefix}/lib/microshift/manifests
+cp -pr ~/manifests $RPM_BUILD_ROOT/%{_prefix}/lib/microshift/
+%clean
+rm -rf $RPM_BUILD_ROOT
+
+%files
+%{_prefix}/lib/microshift/manifests/**
+%changelog
+* <DDD MM DD YYYY username@domain - V major.minor.patch>
+- <your-change-log-comment>
+----
+[.small]
+<1> The `%install` section creates the target directory inside the rpm, `/usr/lib/microshift/manifests/`
+and copies the manifests from the source home directory `~/manifests`.
++
+[IMPORTANT]
+====
+All the required `.yaml` files must be in the  `/usr/lib/microshift/manifests/` directory, including a `kustomize.yaml` if you are using kustomize.
```

Comment:
so we copy from ~/manifests to /usr/.../manifests in this step. So I think it would be better to state that "All ... files must bin in the source ~/manifests", because that is where is copied from and hence the files actually must be.